gui/src/main/java/cern/jarrace/inspector/gui/Stepper.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * © Copyright 2016 CERN. This software is distributed under the terms of the Apache License Version 2.0, copied
+ * verbatim in the file “COPYING”. In applying this licence, CERN does not waive the privileges and immunities granted
+ * to it by virtue of its status as an Intergovernmental Organization or submit itself to any jurisdiction.
+ */
+
+package cern.jarrace.inspector.gui;
+
+import cern.jarrace.commons.domain.AgentContainer;
+import cern.jarrace.inspector.gui.rest.ContainerService;
+import cern.jarrace.inspector.gui.rest.ContainerServices;
+import javafx.application.Application;
+import javafx.application.Platform;
+import javafx.scene.Scene;
+import javafx.scene.layout.FlowPane;
+import javafx.stage.Stage;
+
+/**
+ * A user interface that can list and run {@link AgentContainer}s. The containers are known from a remote HTTP REST
+ * endpoint.
+ */
+public class Stepper extends Application {
+
+    public static final String STEPPER_CSS = "stepper.css";
+
+    private static final ContainerServices containers = ContainerServices.ofBaseUrl("http://localhost:8080/jarrace/");
```

Comment:
Agreed. I could imagine we'd have some way of coding the URL in the GUI like you proposed. But let's cross that bridge when we need to ;)